### ISSUE TITLE
Plugin causes less mixin syntax compilation error 插件导致 less mixin 语法编译报错

### ISSUE THREAD

**Issue by hughfenghen** [mzohaibqc/antd-theme-webpack-plugin#23]
```less
.a,
#b {
  color: red;
}

.mixin-class {
  .a();
}

.mixin-id {
  #b();
}
```
![image](https://user-images.githubusercontent.com/3307051/54257413-2df7b780-459a-11e9-8f4e-3a7507be7ad8.png)

相关issue：https://github.com/ant-design/ant-design-pro/issues/3335

**Comment by mzohaibqc** [comment]
Are you sure this is correct syntax?
```
.a,
#b {
  color: red;
}

.mixin-class {
  .a();
}

.mixin-id {
  #b();
}```

I don't think there is something wrong with this plugin. I can be due to some less version conflict. Not sure but could not tell by looking at this code and error.

**Comment by hughfenghen** [comment]
Code is correct，copy from [less.org](http://lesscss.org/features/#mixins-feature).

I try reproduce error with in `antd-theme-webpack-plugin/examples`, but `yarn run start` in  `create-react-app` throw a error: `Cannot find module react-scripts/config/webpack.config.dev`. 

I try to reproduce the problem on weekends.

Maybe not cause by `antd-theme-webpack-plugin`, because I used `antd-theme-webpack-plugin indirectly with `umi`.

I commented on the following code and the error disappears.
![image](https://user-images.githubusercontent.com/3307051/54369890-a604d100-46b1-11e9-8774-2593883c28d6.png)
